Gemfile

```diff
@@ -53,3 +53,9 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'graphql'
```

Comment:
Thoughts on introducing rubocop to enforce [thoughtbot styles for ruby](https://github.com/thoughtbot/guides/tree/master/style/ruby)? The single quotes made me think of this, since the guides recommend double quotes. It could also be worth thinking about adding [Hound](https://houndci.com) to get style feedback on PRs.